contracts/staking/src/execute.rs

```diff
@@ -131,11 +132,18 @@ fn update_oracle_msgs(deps: Deps, env: Env, config: &Config) -> Result<Vec<Cosmo
         msg: update_redemption_rate_msg_json.as_bytes().to_vec(),
         funds: vec![]
     }.into());
+    messages.push(MsgExecuteContract {
+        sender: env.contract.address.to_string(),
+        contract: config.oracle_contract_address_v2.clone().unwrap().to_string(),
+        msg: update_redemption_rate_msg_json.as_bytes().to_vec(),
+        funds: vec![]
+    }.into());
```

Comment:
I think we don't need to push redemption rate to the new contract.

Reply:
Makes sense. @RiccardoM The `redemption rate` we're providing right now is actually `purchase rate`. So some of partners asked us to provide it as a `purchase rate` and that's why we're trying to migrate contracts. 
We're gonna provide 
for old version -> `redemption rate` (no changes)
for v2 -> `purchase rate`

Reply:
Thanks @rustcandy for the explanation!

Reply:
Oh I didn't mention the reason sorry. Thanks for the explanation